lib/administration/resource/src/resource_administration.cpp

```diff
@@ -191,10 +183,6 @@ namespace irods::experimental::administration::NAMESPACE_IMPL
 
             return std::nullopt;
         }
-        catch (const irods::exception& e) {
-            // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-array-to-pointer-decay)
-            THROW(e.code(), e.what());
-        }
```

Comment:
whups

Reply:
Should I ignore this comment lol?

Reply:
Please do. I felt like this change maybe belongs in the previous commit, but probably not worth the effort to address it.

Reply:
Nice catch. I'll move that change into the previous one.

Reply:
Fixed it.